backend/enums/activity.js

```diff
@@ -1,20 +1,23 @@
-import {convertToPostDto} from "../dto/posts";
+/* eslint-disable import/prefer-default-export */
```

Comment:
Change to a default export - the eslint rule exists for a reason and there's no good reason here to override it.

Reply:
Done